src/emittermixin.js

```diff
@@ -15,6 +15,13 @@
 CKEDITOR.define( [ 'eventinfo', 'utils' ], ( EventInfo, utils ) => {
 	const EmitterMixin = {
 		/**
+		 * Saves how many callbacks has been already added. Does not decrement when callback is removed.
+		 * Used internally as a unique id for a callback.
+		 * @private
+		 */
+		_counter: 0,
```

Comment:
There's too high chance of a conflict with classes private property. Can't we keep this as a private *variable* in this scope? If not, then maybe `_emitterMixinCallbackCounter` :D.